<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlStandaloneAppWizardOptionPage</class>
 <widget class="QWizardPage" name="QmlStandaloneAppWizardOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="verticalSpacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="orientationBehaviorLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Orientation Behavior:</string>
        </property>
        <property name="buddy">
         <cstring>orientationBehaviorComboBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="orientationBehaviorComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="loadDummyDataCheckBox">
        <property name="text">
         <string>Load dummy data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Symbian specific</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="symbianAppIconLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Application Icon (.svg):</string>
        </property>
        <property name="buddy">
         <cstring>symbianAppIconLoadToolButton</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="symbianTargetUid3Label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Target UID3:</string>
        </property>
        <property name="buddy">
         <cstring>symbianTargetUid3LineEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="symbianTargetUid3LineEdit"/>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QCheckBox" name="symbianEnableNetworkChackBox">
        <property name="text">
         <string>Enable network access</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="symbianAppIconPreview">
        <property name="minimumSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>45</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="symbianAppIconLoadToolButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
